pkg/maven/maven.go

```diff
@@ -107,6 +134,146 @@ func Evaluate(options *EvaluateOptions, expression string, command mavenExecRunn
 	return value, nil
 }
 
+// InstallFile installs a maven artifact and its pom into the local maven repository.
+// If "file" is empty, only the pom is installed. "pomFile" must not be empty.
+func InstallFile(file, pomFile string, command mavenExecRunner) error {
+	if len(pomFile) == 0 {
+		return fmt.Errorf("pomFile can't be empty")
+	}
+
+	var defines []string
+	if len(file) > 0 {
+		defines = append(defines, "-Dfile="+file)
+	} else {
+		defines = append(defines, "-Dfile="+pomFile)
+	}
+	defines = append(defines, "-DpomFile="+pomFile)
+	mavenOptionsInstall := ExecuteOptions{
+		Goals:   []string{"install:install-file"},
+		Defines: defines,
+		PomPath: pomFile,
+	}
```

Comment:
Don't the ExecuteOptions here need to specify the fields M2Path, ProjectSettingsFile and GlobalSettingsFile? Why would it install the artifact into the correct m2-folder otherwise?